### PR TITLE
Local dev vscode

### PR DESCRIPTION
**Pull request by dimakis**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (anishasthana) [inline .vscode/launch.json:2]: Remove these comments?

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            "mode": "auto",
+            "program": "${workspaceFolder}/main.go",
+            "args": [],
+            "env": {
+                "KUBECONFIG": "<PATH_TO_KUBECONFIG>"
+            },
+            "showLog": true
+        }
+    ]
+}
```
Comment (anishasthana) [inline .vscode/launch.json:21]: New line

File: config/default/kustomization.yaml [hunk 3]
```diff
@@ -12,10 +12,7 @@ namePrefix: instascale-1-
 #commonLabels:
 #  someName: someValue
 
-bases:
-- ../crd
-- ../rbac
-- ../manager
+# - ../crd
```
Comment (anishasthana) [inline config/default/kustomization.yaml:15]: Remove this bit?

**Comment by anishasthana** [comment]
Changes generally seem good to me. Can we get someone who uses vscode to test this?

**Comment by dimakis** [comment]
@Bobbins228 or @VanillaSpoon have ye used this with vscode?

**Comment by dimakis** [comment]
@anishasthana Can i get an lgtm please?

**Review by anishasthana — APPROVED** [review]

**Pull request merged**